--- monitoring.host.view.html.php	2026-04-03 03:08:05.561377642 +0200
+++ monitoring.host.view.html.php.patched	2026-04-03 03:17:10.032007597 +0200
@@ -29,13 +29,14 @@
 		make_sorting_header(_('Name'), 'name', $data['sort'], $data['sortorder'], $view_url),
 		(new CColHeader(_('Interface'))),
 		(new CColHeader(_('Availability'))),
-		(new CColHeader(_('Tags'))),
-		make_sorting_header(_('Status'), 'status', $data['sort'], $data['sortorder'], $view_url),
+		(new CColHeader(_('Dashboards'))),
 		(new CColHeader(_('Latest data'))),
 		(new CColHeader(_('Problems'))),
 		(new CColHeader(_('Graphs'))),
-		(new CColHeader(_('Dashboards'))),
-		(new CColHeader(_('Web')))
+		(new CColHeader(_('Web'))),
+		(new CColHeader(_('Tags'))),
+		make_sorting_header(_('Status'), 'status', $data['sort'], $data['sortorder'], $view_url)
+		
 	])
 	->setPageNavigation($data['paging']);
 
@@ -108,10 +109,16 @@
 		[$host_name, $maintenance_icon],
 		(new CCol(getHostInterface($interface)))->addClass(ZBX_STYLE_NOWRAP),
 		getHostAvailabilityTable($host['interfaces']),
-		(new CDiv($host['tags']))->addClass(ZBX_STYLE_TAGS_WRAPPER),
-		$host['status'] == HOST_STATUS_MONITORED
-			? (new CSpan(_('Enabled')))->addClass(ZBX_STYLE_GREEN)
-			: (new CSpan(_('Disabled')))->addClass(ZBX_STYLE_RED),
+		$host['dashboards']
+			? [
+				new CLink(_('Dashboards'),
+					(new CUrl('zabbix.php'))
+						->setArgument('action', 'host.dashboard.view')
+						->setArgument('hostid', $host['hostid'])
+				),
+				CViewHelper::showNum($host['dashboards'])
+			]
+			: (new CSpan(_('Dashboards')))->addClass(ZBX_STYLE_DISABLED),
 		[
 			$data['allowed_ui_latest_data']
 				? new CLink(_('Latest data'),
@@ -136,16 +143,6 @@
 				CViewHelper::showNum($host['graphs'])
 			]
 			: (new CSpan(_('Graphs')))->addClass(ZBX_STYLE_DISABLED),
-		$host['dashboards']
-			? [
-				new CLink(_('Dashboards'),
-					(new CUrl('zabbix.php'))
-						->setArgument('action', 'host.dashboard.view')
-						->setArgument('hostid', $host['hostid'])
-				),
-				CViewHelper::showNum($host['dashboards'])
-			]
-			: (new CSpan(_('Dashboards')))->addClass(ZBX_STYLE_DISABLED),
 		$host['httpTests']
 			? [
 				new CLink(_('Web'),
@@ -156,7 +153,11 @@
 				),
 				CViewHelper::showNum($host['httpTests'])
 			]
-			: (new CSpan(_('Web')))->addClass(ZBX_STYLE_DISABLED)
+			: (new CSpan(_('Web')))->addClass(ZBX_STYLE_DISABLED),
+		(new CDiv($host['tags']))->addClass(ZBX_STYLE_TAGS_WRAPPER),
+		$host['status'] == HOST_STATUS_MONITORED
+			? (new CSpan(_('Enabled')))->addClass(ZBX_STYLE_GREEN)
+			: (new CSpan(_('Disabled')))->addClass(ZBX_STYLE_RED)
 	]);
 }
 
